WalletWasabi.Fluent/Extensions/ObservableMixin.cs

```diff
@@ -4,7 +4,7 @@
 
 namespace WalletWasabi.Fluent.Extensions;
 
-public static class ObservableExtension
+public static class ObservableMixin
```

Comment:
OK, I'll rename all of them to be more consistent to use the "Extensions" suffix. It's the most widely used suffix for static classes which only hold extension methods.

Using "Extension" in singular, doesn't look fine, because it intrinsically gives entity to it, and I think that class names of this kind aren't most than containers.

**NOTES**
I used `Mixin` in the past because it's a wider, more abstract concept. I was really used to it, in fact. Some projects use that suffix. 

A Mixin is a class that contains methods for another classes (https://en.wikipedia.org/wiki/Mixin). This is so similar to what extension methods do. 

The concept of Mixin was originated in other languages that support adding extra functionalities to types without modifying them before C# even had extension methods.